pkg/new-ui/v1beta1/backend.go

```diff
@@ -33,6 +33,7 @@ import (
 	experimentv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/experiments/v1beta1"
 	suggestionv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/suggestions/v1beta1"
 	trialsv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/trials/v1beta1"
+	trialv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/trials/v1beta1"
```

Comment:
Do we need this dependency?
Probably, we can use `trialsv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/trials/v1beta1"` above.